os/board/rtl8730e/src/component/wifi/wifi_interactive_mode.c

```diff
@@ -604,7 +605,7 @@ void cmd_wifi_info(int argc, char **argv)
 
 	for (i = 0; i < NET_IF_NUM; i++) {
 		if (wifi_is_running(i)) {
-			nvdbg("\n\r\nWIFI %s Status: Running", ifname[i]);
+			nvdbg("\n\r\nWIFI idx %d Status: Running", i);
```

Comment:
@ZhenBei-Sin Could you review this file change? It looks this file is modified to resolve build break with debug config enabled.

Reply:
Hi @sunghan-chang ,

https://github.com/Samsung/TizenRT/blob/master/os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c#L676
Previously rtl8721csm declared the array ifname[2].
In rtl8730e was removed, the i idx value 0:WLAN0; 1:WLAN1 displays the same information.
LGTM.